modules/objdetect/include/opencv2/objdetect/aruco_board.hpp

```diff
@@ -220,10 +216,16 @@ class CV_EXPORTS_W CharucoBoard : public Board {
      */
     CV_WRAP bool checkCharucoCornersCollinear(InputArray charucoIds) const;
 
-protected:
-    struct CharucoImpl;
-    friend struct CharucoImpl;
-    Ptr<CharucoImpl> charucoImpl;
+    /** @brief Given a board configuration and a set of detected markers, returns the corresponding
+     * image points and object points to call solvePnP
```

Comment:
`#solvePnP`

Reply:
add solvePnP() to doxygen refetence

![image](https://user-images.githubusercontent.com/22337800/209332190-9827ee42-8b40-4f85-a11c-29630b37c604.png)